kokoro/gcp_ubuntu/bazel/build.sh

```diff
@@ -26,12 +28,6 @@ export PS4='[$(date -u "+%T %Z")] '
 bazel --version
 python3 -V
```

Comment:
So we should probably actually be printing PYTHON_BIN here. Or maybe both because of the weird python issue where PYTHON_BIN doesn't dictate everything. Also what about getting version info for CXX and CC as well

```suggestion
"$CXX" --version
"$CC" --version
"$PYTHON_BIN" -V
# TODO( #1875 ): Make PYTHON_BIN also control the runtime version
python3 -V
```